src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/ApprovalCase.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class ApprovalCase(
+  val licenceId: Long?,
+  val name: String?,
+  val prisonerNumber: String?,
+  val submittedByFullName: String?,
+  val releaseDate: LocalDate?,
+  val sortDate: LocalDate?,
```

Comment:
don't need to fix now but think you can get away with just returning releaseDate as a LocalDate as they're the same thing.

One is a date on the frontend for convenience for sorting but can probably sort just on the backend anyway?